examples/with-nextjs-and-clerk-auth/src/themes/monite.ts

```diff
@@ -0,0 +1,171 @@
+import { createTheme } from '@mui/material';
+import type { Components } from '@mui/material/styles/components';
+import type {
+  Palette,
+  PaletteOptions,
+} from '@mui/material/styles/createPalette';
+import type { Theme } from '@mui/material/styles/createTheme';
+import type { TypographyOptions } from '@mui/material/styles/createTypography';
+import { deepmerge } from '@mui/utils';
+import {
+  moniteLight as baseMoniteLight,
+  moniteDark as baseMoniteDark,
+} from '@team-monite/sdk-themes';
+
+const paletteLight: PaletteOptions = {
+  primary: {
+    dark: '#1D59CC',
+    main: '#3737FF',
+    light: '#F4F8FF',
+  },
+  background: {
+    menu: '#F1F2F5',
+    highlight: '#EBEBFF',
+  },
+  neutral: {
+    '10': '#111111',
+    '50': '#707070',
+    '80': '#DDDDDD',
+  },
+};
+
+const paletteDark: PaletteOptions = {
+  primary: {
+    dark: '#1D59CC',
+    main: '#3737FF',
+    light: '#F4F8FF',
+  },
+  background: {
+    menu: '#F1F2F505',
+  },
+  neutral: {
+    '80': '#B8B8B8',
+    '50': '#F3F3F3',
+    '10': '#FFFFFF',
+  },
+};
+
+const typography:
+  | TypographyOptions
+  | ((palette: Palette) => TypographyOptions) = {
+  subtitle2: {
+    fontSize: '1.125rem',
+    fontWeight: 600,
+  },
+  body1: {
+    fontSize: '1rem',
+    fontWeight: 400,
+  },
+  body2: {
+    fontSize: '0.875rem',
+    fontWeight: 500,
+  },
+  label2: {
+    fontSize: '0.875rem',
+    fontStyle: 'normal',
+    fontWeight: 500,
+  },
+  label3: {
+    fontSize: '0.75rem',
+    fontStyle: 'normal',
+    fontWeight: 600,
+  },
+};
+
+const typographyLight = deepmerge(typography, {
+  body2: {
+    color: paletteLight.neutral && paletteLight.neutral['10'],
+  },
+  label3: {
+    color: paletteLight.neutral && paletteLight.neutral['50'],
+  },
+});
+
+const typographyDark = deepmerge(typography, {
+  body2: {
+    color: paletteDark.neutral && paletteDark.neutral['10'],
+  },
+  label3: {
+    color: paletteDark.neutral && paletteDark.neutral['80'],
+  },
+});
+
+const components: Components<Omit<Theme, 'components'>> = {
+  MuiDrawer: {
+    styleOverrides: {
+      root: {
+        '&.navigation-drawer .MuiPaper-root': {
```

Comment:
Maybe naming for the class such as `LayoutNavigationDrawer` would be a better fit. I would prefer a casing like in the MUI.